### DISCUSSION TITLE
Filter/Search organization repos

### DISCUSSION THREAD

**Discussion by JetUni** [community/community#4798] [Mobile]
It would be really wonderful if we were able to filter through the repository of an organization somehow. Because of the microservice architecture, my organization has over 700 repos. There's no way I'm scrolling through those to find the one I'm looking for!

**Comment by thiagohmcruz** [comment]
Same! My org has 6k+ repos so it would be great to be able to search in the "Organizations > SomeOrg > Repositories" list
